lib/exec.js

```diff
@@ -10,7 +10,7 @@ module.exports = function exec(script) {
 
 	script.env = script.env || {};
 
-	var env = objectAssign(process.env, script.env);
+	var env = objectAssign({}, script.env, process.env);
```

Comment:
Also, minor improvement here. It would be nice not to mutate parent process object during assignment for children